Error when problem name contains a path

```julia
julia> model = CUTEstModel("/usr/local/opt/maros_meszaros/share/maros_meszaros/BOYD1.SIF")
ld: can't open output file for writing: lib/usr/local/opt/maros_meszaros/share/maros_meszaros/BOYD1.dylib, errno=2 for architecture x86_64
collect2: error: ld returned 1 exit status
ERROR: failed process: Process(`gfortran -dynamiclib -undefined dynamic_lookup -o lib/usr/local/opt/maros_meszaros/share/maros_meszaros/BOYD1.dylib ELFUN.o EXTER.o GROUP.o RANGE.o /usr/local/opt/cutest/libexec/objects/mac64.osx.gfo/double/libcutest_double.dylib`, ProcessExited(1)) [1]
```
The issue is that the name of the OUTSDIF and AUTOMAT files contain paths that don't exist.

--- a/src/CUTEst.jl
+++ b/src/CUTEst.jl
@@ -79,10 +79,10 @@ Example:
     `sifdecoder("DIXMAANJ", "-param", "M=30")`.
 """
 function sifdecoder(name :: String, args...; verbose :: Bool=false,
-                    outsdif :: String="OUTSDIF_$name.d",
-                    automat :: String="AUTOMAT_$name.d")
+                    outsdif :: String="OUTSDIF_$(basename(name)).d",
+                    automat :: String="AUTOMAT_$(basename(name)).d")
   # TODO: Accept options to pass to sifdecoder.
-  pname, sif = splitext(name);
+  pname, sif = splitext(basename(name));
   libname = "lib$pname";
 
   # work around bogus "ERROR: failed process"
@@ -106,8 +106,8 @@ end
 
 # Initialize problem.
 function CUTEstModel(name :: String, args...; decode :: Bool=true, verbose ::Bool=false)
-  outsdif = "OUTSDIF_$name.d";
-  automat = "AUTOMAT_$name.d";
+  outsdif = "OUTSDIF_$(basename(name)).d";
+  automat = "AUTOMAT_$(basename(name)).d";
   global cutest_instances
   cutest_instances > 0 && error("CUTEst: call finalize on current model first")
   io_err = Cint[0];

